shared/components/CommunityCodeOfConduct/index.js

```diff
@@ -0,0 +1,128 @@
+            Harassing <strong>photography</strong> or <strong>recording</strong>.
+          </li>
+          <li>
+            Sustained <strong>disruption</strong> of talks or other events.
+          </li>
+          <li>
+            Inappropriate <strong>physical contact</strong>.
+          </li>
+          <li>
+            Unwelcome <strong>sexual attention</strong>.
+          </li>
+          <li>
+            Advocating for, or encouraging, any of the above behaviour.
+          </li>
+        </ul>
+        <p>
+          In addition to the above, <strong>no weapons</strong> or <strong>illegal
+          substances</strong> are allowed in any meetup venues at any time.
+        </p>
+      </div>
+    </section>
+
+    <section className="CommunityCodeOfConduct__section block">
+      <div className="content">
+        <h3 className="CommunityCodeOfConduct__section__heading">Enforcement</h3>
+        <p>
+          Participants asked to stop any harassing behavior are expected to comply immediately.
+        </p>
+        <p>
+          If a participant engages in harassing behaviour, event organisers
+          retain the right to take any actions to keep the event a welcoming environment
+          for all participants. This includes <strong>warning the
+          offender</strong> or <strong>expulsion from meetup</strong>.
+        </p>
+        <p>
+          Event organisers may take action to redress anything designed to, or
+          with the clear impact of, disrupting the event or making the environment
+          hostile for any participants.
+        </p>
+      </div>
+    </section>
+
+    <section className="CommunityCodeOfConduct__section block">
+      <div className="content">
+        <h3 className="CommunityCodeOfConduct__section__heading">Reporting</h3>
+        <p>
+          If someone makes you or anyone else feel unsafe or unwelcome, please
+          report it as soon as possible. Harassment and other code of conduct violations reduce the
+          value of our event for everyone.
+        </p>
+        <p>
+          <strong>We want you to be happy at React London events.</strong>
+        </p>
+      </div>
+    </section>
+
+    <section className="CommunityCodeOfConduct__section block">
+      <div className="content">
+        <h3 className="CommunityCodeOfConduct__section__heading">Talk to us</h3>
+        <p>
+          In addition to speaking to us in person, you can also contact us
+          using the following methods:
+        </p>
+        <ul>
+        {/*
+          <li>
+            <strong>Calling</strong> or <strong>messaging</strong> a female:
+            42095823 or a male: 1349084
+          </li>
+        */}
+          <li>
+            <strong>Email us </strong>
+            <a href="mailto:hello@react.london" target="_blank" rel="noopener">
+              hello@react.london
+            </a>
+          </li>
+          <li>
+            <strong>DM us </strong> @reactlondon_
```

Comment:
this isn't very clear - maybe 'direct message us' and `@reactlondon_` be a link to twitter? Also this relies on react london following you, which isn't great...

Reply:
good pick up! just spoke to Laura about this and i'll change it to 'Tweet us' and link the handle to our twitter as well